test/corpus/message.txt

```diff
@@ -90,8 +90,7 @@ closes #123
 (source
   (subject)
   (message
-    (item))
-  (MISSING "source_token1"))
```

Comment:
First, naming `$._newline` would rename `source_token1` to `_newline`.  Second, starting `$._body_line` with `$._newline` allows missing newlines, eliminating this error for an otherwise valid commit message.